src/client/app/services/github.service.ts

```diff
@@ -96,7 +96,7 @@ export class GitHubService {
             return Observable.of([]);
         }
         let url = `${this._baseUrl}/users/${this.profile.login}/gists`;
-        return this._request.get<IGist[]>(url, ResponseTypes.JSON, this._headers);
+        return this._request.get<IGist[]>(url, ResponseTypes.JSON, this._headers, false);
```

Comment:
Add a parameter comment:

    return this._request.get<IGist[]>(url, ResponseTypes.JSON, this._headers, false /*useCache*/);